airflow/config_templates/config.yml

```diff
@@ -1106,6 +1106,17 @@ metrics:
       example: "\"scheduler,executor,dagrun,pool,triggerer,celery\"
         or \"^scheduler,^executor,heartbeat|timeout\""
       default: ""
+    metrics_consistency_on:
+      description: |
+        Enables metrics consistency across all metrics loggers (ex: timer and timing metrics).
+
+        .. warning::
+
+          It is enabled by default from Airflow 3.
+      version_added: 2.10.0
+      type: string
+      example: ~
+      default: "True"
```

Comment:
!!!! @potiuk I guess no one noticed, cos this made it out with a default of on.

Reply:
Oh, except this hasn't made it to a release yet, and it wasn't marked for backporting. (Lucky miss there)

Reply:
Didn't slip through, it was discussed above and intentional.

Reply:
We agreed to enable it directly in Airflow 3 without backporting. However, updating this was overlooked.